podcasts/PlaybackManager.swift

```diff
@@ -418,7 +428,21 @@ class PlaybackManager: ServerPlaybackDelegate {
 
         let currentTime = playingEpisode.playedUpTo
         seekingTo = time
-        FileLog.shared.addMessage("seek to \(time) startPlaybackAfterSeek \(startPlaybackAfterSeek)")
+        if let seekHint {
+            switch seekHint {
+            case .back:
+                print("seek backward")
+                if seekingTo > time {
```

Comment:
@SergioEstevao you were onto something here, but the logic is wrong: we're comparing `seekingTo` and `time` but in line `430` we do: `seekingTo = time`

So this will never succeed.